index.js

```diff
@@ -12,10 +12,10 @@ var regexSequences = [
     // Remove XML stuffs and comments
     [/<\?xml[\s\S]*?>/gi, ""],
     [/<!doctype[\s\S]*?>/gi, ""],
-    [/<!--.*-->/gi, ""],
+    [/<!--[\s\S]*?-->/g, ""],
```

Comment:
Again, not sure if anyone else is rusty but `.` does not match newlines. Adding newlines can be done with `(.|\n)` but that adds a capturing group so `[\s\S]` is used instead, which matches all whitespace and all non-whitespace characters which together form the set of all characters. No casing specifier is needed so that's dropped. `*?` is like `*` but non-greedy.